pkg/monitoring/poller.go

```diff
@@ -75,11 +75,10 @@ func (s *sourcePoller) Run(ctx context.Context) {
 					continue
 				} else if errors.Is(err, context.Canceled) {
 					return
-				} else {
-					s.log.Errorw("failed to fetch from source", "error", err)
-					reusedTimer.Reset(s.pollInterval)
-					continue
 				}
+				s.log.Errorw("failed to fetch from source", "error", err)
```

Comment:
This feels odd to me - the previous formatting was easier to read

Reply:
The idea is to reduce unnecessary indentation.
https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#superfluous-else

Reply:
https://go.dev/doc/effective_go#control-structures
> In the Go libraries, you'll find that when an if statement doesn't flow into the next statement—that is, the body ends in break, continue, goto, or return—the unnecessary else is omitted.